tests/performance_tests.py

```
@@ -9,8 +9,13 @@
 
 import pytest
 
-from py_hamt import HAMT
-from py_hamt.store import InMemoryCAS
+from py_hamt import HAMT, InMemoryCAS, KuboCAS
+import numpy as np
```

Ruff (0.11.9)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)